[EDU-3178] feat: add intelligent dns configure main settings (#515)
aziontech/docs: src/content/docs/en/pages/secure-journey/intelligent-dns/edit-intelligent-dns/edit-main-settings.mdx

---
title: How to configure Intelligent DNS main settings
description: Customize your intelligent DNS domains.
meta_tags: secure, intelligent dns, configuration, settings, domains
namespace: docs_guides_secure_idns_configure_main_settings
permalink: /documentation/products/guides/secure/intelligent-dns-configure-main-settings/
menu_namespace: secureMenu
---

import Button from '~/components/Button.astro'
import Badge from '~/components/Badge.astro'

In this guide, you'll learn to create an [intelligent DNS](/en/documentation/products/intelligent-dns/) zone with its main settings and edit them after you create the zone.

---

## Via RTM

In this section, you'll create or edit the main settings of **Intelligent DNS** with a domain via [Real-Time Manager (RTM)](https://manager.azion.com).

> 1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > select **Intelligent DNS**.

Reviewer: ```suggestion
1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > *Intelligent DNS**.
```

2. To create a new zone, click the **Add Zone** button. To edit an existing zone, select the intelligent DNS you want to edit from the list.

:::caution[warning]

> Once you host your domain on Azions Intelligent DNS, you must change your domain's authoritative DNS servers to use Azion's namservers:

Reviewer 2: ```suggestion
Once you host your domain on Azion's Intelligent DNS, you must change your domain's authoritative DNS servers to use Azion's nameservers:
```

Reviewer 2: I had suggested Azion's Intelligent, but I saw later on that you used Azion Intelligent... so

- ns1.aziondns.net
- ns2.aziondns.com
- ns3.aziondns.org
:::

> ## Creating a new zone

Reviewer: ```suggestion
### Creating a new zone
```

1. Give your intelligent DNS a unique and easy-to-remember name.

> 2. In **Domain**, add the name of the domain you want to host in Intelligent DNS.

Author: ```suggestion
2. In **Domain**, add the name of the domain you want to host in Intelligent DNS. For example: `mydomain`.
```

<Button href="/en/documentation/products/guides/secure/add-records/" text="go to add records guide" variant="secondary"></Button>

---

## Via API

1. Run the following `POST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) to create a new intelligent DNS:

```bash
curl --location 'https://api.azionapi.net/intelligent_dns' \
--header 'Authorization: Token [TOKEN VALUE]' \
--header 'Accept: application/json; version=3' \
--header 'Content-Type: application/json' \
--data '{
    "name": "New zone",
    "domain": "yourdomain.com",
    "is_active": true
}'
```

2. You'll receive a response similar to this:

```json
{
  "results": [
    {
      "nx_ttl": 3600,
      "domain": "yourdomain.com",
      "retry": 7200,
      "name": "New zone",
      "nameservers": [
        "ns1.aziondns.net",
        "ns2.aziondns.com",
        "ns3.aziondns.org"
      ],
      "soa_ttl": 3600,
      "is_active": true,
      "refresh": 43200,
      "expiry": 1209600,
      "id": 1234
    }
  ],
  "schema_version": 3
}
```

:::caution[warning]

> Once you host your domain on Azion Intelligent DNS, you must change your domain's authoritative DNS servers to use Azion's namservers:

Reviewer 2: ```suggestion
Once you host your domain on Azion Intelligent DNS, you must change your domain's authoritative DNS servers to use Azion's nameservers:
```
